base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -130,7 +133,7 @@ impl<TBackend, TWalletConnectivity, TKeyManagerInterface>
 where
     TBackend: OutputManagerBackend + 'static,
     TWalletConnectivity: WalletConnectivityInterface,
-    TKeyManagerInterface: TransactionKeyManagerInterface,
+    TKeyManagerInterface: SecretTransactionKeyManagerInterface,
```

Comment:
```suggestion
    TKeyManagerInterface: TransactionKeyManagerInterface,
```
Dont include this trait. You are not suppose to expose the private keys here